### PR TITLE
If env is debug display line and filename

### PR DESCRIPTION
**Pull request by cylwin**
display filename only if lunch with debug=true

### CODE REVIEW

File: main.coffee [hunk 1]
```diff
@@ -31,7 +31,21 @@ class Logger
             text = JSON.stringify text
         return text
 
+    getFileAndLine: ->
+        stacklist = (new Error()).stack.split('\n').slice(3);
+        stackReg = /at\s+(.*)\s+\((.*):(\d*):(\d*)\)/gi;
+        stackReg2 = /at\s+()(.*):(\d*):(\d*)/gi;
+
+        s = stacklist[0]
```
Comment (frankrousseau) [inline main.coffee]: Please use only explicit variable names.

Comment (frankrousseau) [inline main.coffee]: @cylwin Did you find time to fix that?

Comment (cylwin) [inline main.coffee]: I totally forgot this, I'm doing it now

**Comment by frankrousseau** [comment]
The variable name like `s` or `sp` are still not readable. Can you rename them?

**Comment by clochix** [comment]
Thanks for the review @frankrousseau :-)
@cylwin could you address this comments, so we could merge this one ? Thanks !

**Comment by cylwin** [comment]
Now, it seems to be ok.

**Comment by clochix** [comment]
Thanks @cylwin :-)
@frankrousseau or @aenario would you mind merging this ?

**Pull request merged**